tests/v3/test_codecs/test_codecs.py

```diff
@@ -255,90 +238,6 @@ async def test_delete_empty_chunks(store: Store) -> None:
     assert await store.get(f"{path}/c0/0", prototype=default_buffer_prototype()) is None
 
 
-@pytest.mark.parametrize("store", ("local", "memory"), indirect=["store"])
-async def test_zarr_compat(store: Store) -> None:
```

Comment:
drawing attention to this change which we added to develop confidence that we were doing backward compatible things. 

We may consider replacing this with a new CI workflow / test suite that generates data w/ 2.18 and consumes it with the v3 branch.

Reply:
Is it worth opening an issue to keep track of replacing this?

Reply:
https://github.com/zarr-developers/zarr-python/issues/2185